docs/experiments-tct_colbert-v2.md

```diff
@@ -0,0 +1,200 @@
+# Pyserini: Reproducing TCT-ColBERT-V2 Results
+
+This guide provides instructions to reproduce the TCT-ColBERT-V2 dense retrieval model described in the following paper:
+
+> Sheng-Chieh Lin, Jheng-Hong Yang, and Jimmy Lin. [In-Batch Negatives for Knowledge Distillation with Tightly-CoupledTeachers for Dense Retrieval.]_RepL4NLP 2021_.
+
+Since dense retrieval depends on neural networks, Pyserini requires a more complex set of dependencies to use this feature.
+See [package installation notes](../README.md#package-installation) for more details.
+
+Note that we have observed minor differences in scores between different computing environments (e.g., Linux vs. macOS).
+However, the differences usually appear in the fifth digit after the decimal point, and do not appear to be a cause for concern from a reproducibility perspective.
+Thus, while the scoring script provides results to much higher precision, we have intentionally rounded to four digits after the decimal point.
+
+## MS MARCO Passage Ranking
+
+Summary of results:
+
+| Condition | MRR@10 (paper) | MAP | Recall@1000 |
+|:----------|-------:|----:|------------:|
+| TCT_ColBERT-V2 (brute-force index) |  0.3439 (0.3435) | 0.3509 | 0.9670 |
+| TCT_ColBERT-V2-HN (brute-force index) |  0.3542 (0.3540) | 0.3608 | 0.9708 |
+| TCT_ColBERT-V2-HN+ (brute-force index) | 0.3584 (0.3586) | 0.3645 | 0.9695 |
+| TCT_ColBERT-V2-HN+ (brute-force index) + BoW BM25 | 0.3683 (0.3687)  | 0.3738 | 0.9707 |
+| TCT_ColBERT-V2-HN+ (brute-force index) + BM25 w/ doc2query-T5 | 0.3730 (0.3747) | 0.3789 | 0.9759 |
+
+Here we notice slight difference between our paper (TF) and reproduction (PT). 
```

Comment:
Can you clarify this? which number is from the paper, which is reprod?

e.g., "figures from the paper are in parentheses"